myosuite/envs/myo/assets/leg/myoosl_control.py

```diff
@@ -0,0 +1,348 @@
+""" =================================================
+# Copyright (c) MyoSuite Authors
```

Comment:
@elladyr - harmonize with other headers

Reply:
What is the newer license that I should use? The previous files (e.g. `env_base`) were using `Apache License, Version 2.0 (the "License")`

Reply:
The licenses are not consistent across the repo. Either we make them consistent now or start a new issue to track this.